Northstar.CustomServers/mod/scripts/vscripts/item_inventory/sv_item_inventory.gnut

```diff
@@ -152,7 +161,6 @@ void function PlayerInventory_TakeAllInventoryItems( entity player )
 
 void function PlayerInventory_RefreshEquippedState( entity player )
 {
-
 }
```

Comment:
oh my god github suggestion why did you delete the line instead of just the whitespace